packages/node_modules/@ciscospark/internal-plugin-lyra/test/integration/spec/space.js

```diff
@@ -156,6 +156,7 @@ describe('plugin-lyra', function () {
         .then(() => spock.spark.internal.lyra.space.getCurrentBindings(lyraMachine.space))
         .then(({bindings}) => assert.lengthOf(bindings, 1)));
 
+      // Does not work on IE. We have no plans to support it.
```

Comment:
~[img](https://media3.giphy.com/media/ws4pzeC916KRO/giphy.gif?cid=e1bb72ff5c19387e345776326bc0179f) lol

Reply:
🤣 Sorry, I spoke with Olivier and he said we do not support it so I focused on the *how* do we smooth this out for contributors.